kubernetes/rollup-chart/files/scripts/start-conductor.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+# request token from http://celestia-token.astria.localdev.me/ and save to SequencerRelayerConfig.toml
+
+# FIXME - how to use `token-svc` port here instead of hardcoding?
+BEARER_TOKEN=$(wget -qO- http://celestia-service:5353)
+
+if [ -z "$BEARER_TOKEN" ]; then
+    echo "Failed to fetch the Celestia bearer token."
+    exit 1
+fi
+
+echo "Celestia Bearer token fetched successfully."
+
+export ASTRIA_celestia_bearer_token="$BEARER_TOKEN"
+
+# TODO - use $gossipnet_port instead of hardcoded 33900
+/usr/local/bin/astria-conductor \
+  --tendermint-url=$cometbft_rpc_endpoint \
+  --celestia-node-url=$celestia_node_url \
+  --chain-id=$evm_chain_id \
+  --execution-rpc-url=http://localhost:$executor_host_grpc_port \
+  --bootnodes=/ip4/192.168.65.120/tcp/33900/p2p/12D3KooWJGy9JbZyi4JLF2PsBsuUm8Jn72qrHiQ5it5wygAAvHYb
```

Comment:
@joroshiba should this be 2451? otherwise, I don't think `$gossipnet_port` is actually being used.

Reply:
This is connecting to the sequencer relayer, so the bootnodes are telling it "here is the node to connect to".  It will communicate it's open port when it connects, so we need the port to be open at 2451, and it needs to dial out. Otherwise the services don't know how to find each other.

Also annoyingly this requires the a fixed IP for the bootnode communication.

Reply:
We need to pass in `--libp2p-port=$gossipnet_port` here in that case